chainercv/links/model/faster_rcnn/faster_rcnn.py

```diff
@@ -270,22 +270,21 @@ def predict(self, imgs):
 
         """
         prepared_imgs = list()
-        scales = list()
+        shapes = list()
```

Comment:
Can you use `sizes` instead of `shapes`?
`size` is used specifically to refer to `H, W`, whereas `shape` is a general tuple.

Also, `shape` should be changed to `size`.

Reply:
I think `size` is not good because of `np.array.size`

Reply:
`shapes` sounds like `(C, H, W)`.
It is less likely that people mix up with `np.array.size`.
For consistency reasons, `size` is better.
(e.g. https://github.com/chainer/chainercv/blob/master/chainercv/transforms/image/resize.py#L44)

Reply:
ok